content/en-us/reference/engine/classes/TweenBase.yaml

```diff
@@ -76,6 +76,11 @@ methods:
 
       If you want to reset the progress variables of the tween, use
       `Class.TweenBase:Cancel()`.
+
+      You can only pause tweens that are in the `Class.TweenBase.PlaybackState|PlaybackState`
+      of `Enum. PlaybackState.Playing`; tweens in other states won't pause.
+      To keep track of the current `PlaybackState`, you can listen to the `PlaybackState`
+      property change events.
```

Comment:
```suggestion
      This may be a factor when a tween is in a different
      `Class.TweenBase.PlaybackState|PlaybackState` such as `Enum. PlaybackState.Delayed`
      as a result of its `Datatype.TweenInfo.DelayTime` being greater than 0; in this case,
      attempting to pause the tween will fail and the tween will play following its specified
      delay time.
```

Reply:
How about this instead:

If a tween is in a different
      `Class.TweenBase.PlaybackState|PlaybackState` such as `Enum. PlaybackState.Delayed`
      (as a result of its `Datatype.TweenInfo.DelayTime` being greater than 0),
      attempting to pause the tween will fail and the tween will play following its specified
delay time.

Reply:
Sounds good!